client/app/lib/providers/managed/addmanagedmachinemodal.coffee

```diff
@@ -13,7 +13,7 @@ module.exports = class AddManagedMachineModal extends ContentModal
   constructor: (options = {}, data) ->
 
     options.cssClass = 'add-managed-vm content-modal'
-    options.title    = 'Add Your Own Machine'
+    options.title    = 'Add Connected Machine'
```

Comment:
oops sorry, there is a misunderstanding here, I will convert it to `Add Your Own Machine`